www/css/dark.css

```diff
@@ -14,25 +15,21 @@ body {
   background-color: var(--bg-color);
 }
 
-.blocklyWorkspace{
+.blocklyWorkspace {
   color: var(--txt-color);
   background-color: var(--bg-color);
 }
 
```

Comment:
The scroll bars are waaay too bright

![image](https://user-images.githubusercontent.com/32238504/62171848-1480bd80-b339-11e9-9dc5-05c26d0d8581.png)

Reply:
![Bars](https://user-images.githubusercontent.com/33164598/62176813-fa76b900-b2ff-11e9-86d4-02efdf8ce754.gif)